package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Feb 21 11:10:24 UTC 2020 - José Iván López González <jlopez@suse.com>
+
+- Delegate the logic for calculating a device udev link to
+  yast2-storage-ng module.
```

Comment:
issue number?

Reply:
No issue number. This is an internal change addressed by ourselves. There is no real bug associated to it, but only a code improvement. I guess we don't need a bug number for that, right?

Reply:
it won't be submitted without it.